src/element/textWysiwyg.tsx

```diff
@@ -123,7 +123,7 @@ export const textWysiwyg = ({
       handleSubmit();
     } else if (event.key === KEYS.ENTER && event[KEYS.CTRL_OR_CMD]) {
       event.preventDefault();
-      if (event.isComposing || event.keyCode === 229) {
```

Comment:
why was this added  ?

Reply:
It was introduced in https://github.com/excalidraw/excalidraw/pull/1185 — I believe it has something to do with IME (can you confirm @haikyuu?). Let's not remove this until we're sure. Can you revert @lipis?